actionpack/lib/action_dispatch/middleware/templates/rescues/diagnostics.html.erb

```diff
@@ -9,7 +9,11 @@
 
 <div id="container">
   <h2><%= h @exception.message %></h2>
-
+  <%- if h @exception.message.to_s.match(/^Migrations are pending.*RAILS_ENV=development$/) %>
+      <form method="post" action="/rails/migrate">
+        <h2> <input value="Attempt Migrating from here" type="submit"/> </h2>
```

Comment:
Why do we want to say `Attempt`? Can't we make our heuristic better or something else going on?